packages/hash/api/src/graphql/typeDefs/ontology/data-type.typedef.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from "apollo-server-express";
+
+export const dataTypeTypedef = gql`
+  scalar DataType
+
+  type IdentifiedDataType {
```

Comment:
This came up in a prior PR, but I think it's worth reflagging as the naming of the GQL type defs is going to be a lot more important than other variables.

Maybe my understanding is incorrect, but to me `IdentifiedDataType` doesn't convey any more meaning than `DataType`. Perhaps you mean `IdentifiableDataType`, which would suggest to me that this is a structure that can be used to re-identify a schema? But that's not actually true since the schema's `$id` can be used instead of the `dataTypeVersionedUri` field.

An idea I have is that in the context of the workspace GQL API (which will only be consumed by the FE), we refer to `DataType` as the thing that gets passed around, and use `DataTypeSchema` to refer to the schema field in the `DataType`. This makes the difference between the types more clear from the naming. I recall that we wanted to avoid doing this though, so perhaps coming up with something else to replace `IdentifiedDataType` is the way to go. 

(cc @Alfred-Mountfield)

Reply:
Sorry for bringing back that term again! I think what I wanted with the `Identified` part was to convey that the schema had "full" metadata associated with the schema, and while I think `DataTypeSchema` could be clear to users of the GraphQL API, the `DataType` itself is the schema. I don't know what a good name would be

Reply:
We've discussed this off-GitHub and agreed to make use of the existing term `Persisted*Type`. We would like to find a better name, but for now the consistency across the stack is important for us

Reply:
Addressed in f4d46d43482d9b1f5a4ebbf268c876177a62aff1